ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -891,21 +916,30 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
             <div style={{...styles.header, color: colors.primary, fontSize: 14, marginBottom: '0.2rem'}}>
               Select account
             </div>
-            <Dropdown style={{width: '14rem'}}
-                      value={this.state.workspace.billingAccountName}
-                      options={this.buildBillingAccountOptions()}
-                      onChange={e => {
-                        if (e.value === CREATE_BILLING_ACCOUNT_OPTION_VALUE) {
-                          this.setState({
-                            showCreateBillingAccountModal: true
-                          });
-                        } else {
-                          this.setState(fp.set(['workspace', 'billingAccountName'], e.value));
-                        }
-                      }}
-            />
-          </WorkspaceEditSection>
-        }
+            <OverlayPanel ref={(me) => freeTierBalancePanel = me} dismissable={true} appendTo={document.body}>
+              <div style={styles.freeCreditsBalanceOverlay}>
+                FREE CREDIT BALANCE {renderUSD(freeTierCreditsBalance)}
+              </div>
+            </OverlayPanel>
+            <FlexRow>
+              <Dropdown style={{width: '14rem'}}
+                        value={this.state.workspace.billingAccountName}
+                        options={this.buildBillingAccountOptions()}
+                        onChange={e => {
+                          if (e.value === CREATE_BILLING_ACCOUNT_OPTION_VALUE) {
+                            this.setState({
+                              showCreateBillingAccountModal: true
+                            });
+                          } else {
+                            this.setState(fp.set(['workspace', 'billingAccountName'], e.value));
+                          }
+                        }}
+              />
+              {freeTierCreditsBalance > 0.0 && <div style={styles.freeCreditsBalanceClickable}>
```

Comment:
Have to add the text padding here because we want the Clickable to be exactly the size of the text.

This is because the Overlay aligns to the Clickable.